ext/opcache/ZendAccelerator.c

```diff
@@ -3491,6 +3529,10 @@ static void preload_link(void)
 					if (!found) continue;
 				}
 
+				if (!preload_all_types_known(ce)) {
```

Comment:
This check is seriously more restricted than necessary, but we try to improve it later. Please add TODO comments.